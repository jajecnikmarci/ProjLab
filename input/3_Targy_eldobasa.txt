starttest
add room room1 3
add room room2 3 poisonous
add item room1 FFP2 ffp2
add item room1 Transistor transistor
add item room1 Camembert camembert
add item room2 Transistor transistor2
add door room1 room2 door1
add person room1 Student student1
pickup student1
pickup student1 
printstate room1
printstate student1
drop student1 camembert
printstate room1
printstate student1
gotoroom student1 room2
pickup student1
printstate room2
printstate student1
endtest