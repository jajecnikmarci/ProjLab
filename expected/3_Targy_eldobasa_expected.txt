room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
		ffp2
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
		camembert
		transistor
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
		ffp2
		camembert
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
		transistor
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room2
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
		transistor2
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: true
	- inventory: 
		transistor
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
